# Do not busy wait for confirmation after recording. [dk]
#PATCHOPTIONS: -p0

--- bplaysrc/shmbuf.c.orig
+++ bplaysrc/shmbuf.c
@@ -443,7 +443,7 @@ printw(" %d",cbuff);
 	cleanupsems();
 
 #ifdef VUMETER
-	nodelay(stdscr, TRUE);
+	nodelay(stdscr, FALSE);
 
 	/* child has displayed informative data */
 	do
